.adaptable/config.schema.json

```diff
@@ -111,7 +111,7 @@
             "tags": ["nodejs"]
         },
         "nodeVersion": {
-            "enum": ["12", "14", "16", "18"],
+            "enum": ["12", "14", "16", "18", "20"],
```

Comment:
This is necessary, but probably not sufficient. This will allow Node 20 with Paketo and produce a difficult to find/understand error when someone changes their existing Paketo project to 20.

Suggest detecting the invalid combination and giving a clear error with info on what to change to fix the error.

Reply:
I think that is over engineering, the GUI Settings limit the choices based on builderType, though the user could always type 20 in the YAML.  I can fix it if you really think it is worthwhile.